datadog/setupdatadog.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+#Set the enviroment for TORQHOME
+cd ..
+source setenv.sh
+
+if [ -z $1 ]; then
+  echo "no argument provided; defaulting dogstatsd_port to 8125"
+  dogstatsd_port=8125
+else
+  dogstatsd_port=$1
+fi
+
+ddconfigfile=${KDBAPPCONFIG}/ddconfig.txt
+echo "dogstatsd_port:"$dogstatsd_port > $ddconfigfile
+
+#File path of the process.yaml file
+filename1=/etc/datadog-agent/conf.d/process.d/process.yaml
```

Comment:
`filename1` is not a very descriptive variable name, should be renamed to something more appropriate.